test/functional/test_framework/wallet.py

```diff
@@ -212,16 +212,18 @@ def get_address(self):
         assert_equal(self._mode, MiniWalletMode.ADDRESS_OP_TRUE)
         return self._address
 
-    def get_utxo(self, *, txid: str = '', vout: Optional[int] = None, mark_as_spent=True) -> dict:
+    def get_utxo(self, *, txid: str = '', vout: Optional[int] = None, mark_as_spent=True, include_immature_coinbase=False) -> dict:
         """
         Returns a utxo and marks it as spent (pops it from the internal list)
 
         Args:
         txid: get the first utxo we find from a specific transaction
         """
         self._utxos = sorted(self._utxos, key=lambda k: (k['value'], -k['height']))  # Put the largest utxo last
+        if not include_immature_coinbase:
+            utxo_filter: Any = filter(lambda utxo: not utxo['coinbase'] or COINBASE_MATURITY <= utxo['confirmations'], self._utxos)
```

Comment:
What's the purpose of this annotation `Any` here?

Reply:
removing `Any` it throw a lint error, I believe it's because it can be any value from `filter` or `reversed`.